extensions/GitExtension/FileExplorerGitIntegration/Models/CommitLogCache.cs

```diff
@@ -94,7 +94,20 @@ public CommitLogCache(Repository repo)
 
     private CommitWrapper? FindLastCommitUsingCommandLine(string relativePath)
     {
-        var result = GitExecute.ExecuteGitCommand(_gitDetect.GitConfiguration.ReadInstallPath(), _workingDirectory, $"log -n 1 --pretty=format:%s%n%an%n%ae%n%aI%n%H -- {relativePath}");
+        if (relativePath == string.Empty)

```

Comment:
should a null check also be added here? Thanks!

Reply:
`relativePath` is not nullable (parameter type is `string`, not `string?`).